website/pages/docs/deployment/ecs.md

```diff
@@ -78,189 +62,227 @@ Once you have a CloudQuery API Key you are going to create a Secret in AWS Secre
-            "Effect": "Allow"
-        },
-        {
-            "Action": [
-                "s3:GetObject"
-            ],
-            "Effect": "Deny",
-            "NotResource": [
-                "arn:aws:s3:::<REPLACE_WITH_S3_DESTINATION_BUCKET>/*"
-            ]
-        },
-        {
-            "Action": [
-                "cloudformation:GetTemplate",
-                "dynamodb:GetItem",
-                "dynamodb:BatchGetItem",
-                "dynamodb:Query",
-                "dynamodb:Scan",
-                "ec2:GetConsoleOutput",
-                "ec2:GetConsoleScreenshot",
-                "ecr:BatchGetImage",
-                "ecr:GetAuthorizationToken",
-                "ecr:GetDownloadUrlForLayer",
-                "kinesis:Get*",
-                "lambda:GetFunction",
-                "logs:GetLogEvents",
-                "sdb:Select*",
-                "sqs:ReceiveMessage"
-            ],
-            "Resource": "*",
-            "Effect": "Deny"
-        }
-    ]
-}
+```yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Deploy CloudQuery on AWS ECS with Fargate'
+
+Parameters:
+  CQVersion:
+    Description: Please enter the version of CloudQuery you want to deploy. This should be in the format  X, X.Y, X.Y.Z, or `latest`
+    Type: String
+  ScheduleExpression:
+    Description: Please enter the Eventbridge Schedule Expression. This can either be a Rate or a cron expression. This will define how often CloudQuery will run the sync.
+    Type: String
+    Default: "rate(24 hours)"
+  PrivateSubnetIds:
+    Description: Please enter a comma separated list of Subnet Ids where you want to run CloudQuery and the Database.
+    Type: String
+  SecurityGroupIds:
+    Description: Please enter a comma separated list of Security Group IDs that you want to attach to the Fargate node.
+    Type: String   
+  DestinationS3Bucket:
+    Description: Please enter the name of the S3 bucket where you want to store the CloudQuery results
+    Type: String
+  CQConfiguration:
+    Description: Please enter the CloudQuery configuration file encoded in base64
+    Type: String
+  CQApiKey:
+    Description: ARN of the secret containing the CloudQuery API Key
+    Type: String
+  AWSMarketplace:
+    Description: If you are using the AWS Marketplace version of CloudQuery, set this to true
+    Type: String
+    AllowedValues: [true, false]
+    Default: false
+Resources:
+  #### ECS Cluster:
+  ECSCluster:
+    Type: 'AWS::ECS::Cluster'
+  
+  ScheduledWorkerTask:
+    Type: 'AWS::ECS::TaskDefinition'
+    Properties:
+      RequiresCompatibilities:
+        - FARGATE
+      NetworkMode: awsvpc
+      Cpu: 1024
+      Memory: 2GB
+      ExecutionRoleArn: !GetAtt ExecutionRole.Arn
+      TaskRoleArn: !GetAtt ExecutionRole.Arn
+      ContainerDefinitions:
+        - Essential: 'true'
+          Command: 
+            - "echo $CQ_CONFIG| base64 -d  > ./file.yml;/app/cloudquery sync ./file.yml --log-console --log-format json"
```

Comment:
I think `base64 -d` is Linux only (`-D` on Mac I think); is this cloudformation template only applied on Linux instances?

Reply:
This is only Linux